### PR TITLE
Remove slash from const name in transactions documentation

### PR DESCRIPTION
**Pull request by greeflas**
|      Q       |   A
|------------- | -----------
| Type         | bug
| BC Break     | no
| Fixed issues | -

#### Summary

Hi!
I have just removed slash from const name to make it look better.

Снимок экрана 2020-08-08 в 00 43 35

### CODE REVIEW

File: docs/en/reference/transactions.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ constants:
 
 The default transaction isolation level of a
 ``Doctrine\DBAL\Connection`` is chosen by the underlying platform
-but it is always at least READ\_COMMITTED.
+but it is always at least READ_COMMITTED.
```
Comment (greg0ire) [inline docs/en/reference/transactions.rst:53]: We can make it look even better:
```suggestion
but it is always at least ``READ_COMMITTED``.
```

By the way, why are you contributing this to `master`? Are other branches unaffected?

Comment (greeflas) [reply docs/en/reference/transactions.rst]: Thanks! All branches are affected so where I should contribute? To each branch separately?

Comment (greg0ire) [reply docs/en/reference/transactions.rst]: You should contribute to the default branch: 2.10.x, we will then merge this up. `git rebase --onto 2.10.x master fix-transactions-doc` is your friend :)

Comment (greeflas) [reply docs/en/reference/transactions.rst]: Done, thank you!

**Comment by greg0ire** [comment]
Both Travis and AppVeyor look like they still run with the configuration for branch master, which I would understand if you hadn't rebased properly, but you very much did. Maybe it did not pick up on the fact that you retargeted? Let's close and reopen.

**Review by greg0ire — DISMISSED** [review]

File: docs/en/reference/transactions.rst [hunk 2]
```diff
@@ -258,4 +258,3 @@ If you need stricter control, you can catch the concrete exceptions directly:
 - ``Doctrine\DBAL\Exception\LockWaitTimeoutException``: this exception happens when
   a transaction has to wait a considerable amount of time to obtain a lock, even if
   a deadlock is not involved.
-
```
Comment (SenseException) [inline docs/en/reference/transactions.rst:261]: Please don't remove the last new line at the end of the file.

**Review by SenseException — CHANGES_REQUESTED** [review]

File: docs/en/reference/transactions.rst (hunk 2 above)
Comment (greeflas) [reply docs/en/reference/transactions.rst]: Fixed.

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by SenseException** [comment]
Thank you @greeflas.

**Pull request merged**